virt/virt-4-11-release-notes.adoc

```diff
@@ -48,6 +48,7 @@ The SVVP Certification applies to:
 //CNV-15218 RHEL 9 is a supported OS
 
 //CNV-18162 Move MTV action from VM list page
+* The link for installing the Migration Toolkit for Virtualization (MTV) Operator has been moved from the *Virtual Machines* page to the Related Operators section of the *Getting started resources* card on the *Virtualization Overview* page of the {VirtProductName} console. If the MTV Operator is installed, a link that launches the MTV console is displayed. 
```

Comment:
The first sentence reads too long, might be better to break it up into something like this:
```
The link for installing the Migration Toolkit for Virtualization (MTV) Operator in the {product-title} web console has been moved. It is now located in the *Related operators* section of the *Getting started resources* card on the *Virtualization Overview* page.
````